src/main/java/seedu/address/commons/core/Messages.java

```diff
@@ -15,10 +15,13 @@
     public static final String MESSAGE_INVALID_FLAG = "Flag is invalid!";
     public static final String MESSAGE_NO_FLAG = "No flag is found!";
     public static final String MESSAGE_PERSONS_LISTED_OVERVIEW = "%1$d persons listed!";
-    public static final String MESSAGE_INTERIVEW_CANNOT_BE_PASSED = "The interview cannot be passed, "
-            + "as the number of current offers will exceed the number of available positions!";
-    public static final String MESSAGE_INTERIVEW_CANNOT_BE_ACCEPTED = "The interview cannot be accepted, "
-            + "as it has not been passed yet!";
-    public static final String MESSAGE_INTERIVEW_CANNOT_BE_REJECTED = "The interview cannot be rejected, "
-            + "as it has not been passed yet!";
+    public static final String MESSAGE_INTERVIEW_CANNOT_BE_PASSED = "The interview cannot be passed, "
+            + "as the number of current offers will exceed the number of available positions or because the status"
+            + " has been modified and it is not pending";
+    public static final String MESSAGE_INTERVIEW_CANNOT_BE_FAILED = "The interview cannot be failed, "
+            + "since it's status has already been modified and it is not pending";
```

Comment:
Similar to above, maybe can be something like `Only pending interviews can be failed` or `You cannot fail an interview that has already been passed`

Reply:
I'll adopt the first one! The second suggestion would be problematic since this is the default error message would appear if the status is `accepted` or `rejected` which is one status ahead of this stage. Thanks @likeabowx !